SPECS/onefuzz/onefuzz.spec

```diff
@@ -0,0 +1,54 @@
+Summary:        A self-hosted Fuzzing-As-A-Service platform
+Name:           onefuzz
+Version:        8.5.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://github.com/microsoft/OneFuzz
+Source0:        https://github.com/microsoft/onefuzz/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Patch0:         git-version.patch
+%define debug_package %{nil}
```

Comment:
This is a shame, I assume it's because release cargo builds by default don't include debugggin symbols?

Reply:
I added that as a package I was copying from included it, I will remove and check it still builds.